### PR TITLE
fix(discover) Finalize available functions

### PR DESCRIPTION
**Pull request by evanh**
We finalized the list of available functions, so reflect that change in the
backend. We still need the discover shim for the old queries, but there
is another PR incoming to migrate all of that over.

### CODE REVIEW

**Review by dashed — APPROVED** [review]
Looks good to me 👍

File: src/sentry/snuba/discover.py [hunk 1]
```diff
@@ -274,13 +274,13 @@ def get_row(row):
...
-    "p99": "percentile(transaction.duration, 0.99)",
+    "p75": "p75()",
+    "p95": "p95()",
+    "p99": "p99()",
     "last_seen": "last_seen()",
     "latest_event": "latest_event()",
-    "apdex": "apdex(transaction.duration, 300)",
-    "impact": "impact(transaction.duration, 300)",
+    "apdex": "apdex(300)",
+    "impact": "impact(300)",
```
Comment (markstory) [inline src/sentry/snuba/discover.py]: Should this list be removed or are you planning to do that with the migrations changes?

Comment (evanh) [reply src/sentry/snuba/discover.py]: This has to be removed after the migrations, so in a separate PR, once we're sure no one is using the old aliases anymore.

Comment (markstory) [reply src/sentry/snuba/discover.py]: Sounds good. I'll clean up the frontend code post migration too then.

**Review by markstory — APPROVED** [review]

**Review by markstory — APPROVED** [review]

**Pull request merged**